src/bin/polymer-bundler.ts

```diff
@@ -0,0 +1,178 @@
+        'Takes an argument in the form of URI|PATH where url is a URI composed of a protocol, hostname, and path and PATH is a local filesystem path to replace the matched URI part with. Multiple redirects may be specified; the earliest ones have the highest priority.'
+  },
+  {
+    name: 'add-import',
+    type: String,
+    multiple: true,
+    description:
+        'Add this import to the target HTML before vulcanizing. Can be used multiple times',
+    typeLabel: `${pathArgument}`
+  },
+  {
+    name: 'strip-exclude',
+    type: String,
+    multiple: true,
+    description: 'Exclude a subpath and strip the link that includes it',
+    typeLabel: `${pathArgument}`
+  },
+  {
+    name: 'strip-comments',
+    type: Boolean,
+    description: 'Strips all HTML comments not containing an @license from the document'
+  },
+  {
+    name: 'no-implicit-strip',
+    type: Boolean,
+    description: 'DANGEROUS! Avoid stripping imports of the transitive dependencies of imports specified with `--exclude`. May result in duplicate javascript inlining.'},
+  {
+    name: 'inline-scripts',
+    type: Boolean,
+    description: 'Inline external scripts'
+  },
+  {
+    name: 'inline-css',
+    type: Boolean,
+    description: 'Inline external stylesheets'
+  },
+  {
+    name: 'out-html',
+    type: String,
+    description: `If specified, output will be written to ${pathArgument} instead of stdout.`,
+    typeLabel: `${pathArgument}`
+  },
+  {name: 'in-html', type: String, defaultOption: true, description: 'Input HTML. If not specified, will be the last command line argument.'},
+];
+
+const usage = [
+  {header: 'Usage', content: ["vulcanize [options...] <in-html>"]}
+  {header: 'Options', optionList: optionDefinitions},
+  {header: 'Examples', content: `  The command
+    vulcanize target.html
+  will inline the HTML Imports of \`target.html\` and print the resulting HTML to standard output.
+
+  The command
+    vulcanize target.html > build.html
+  will inline the HTML Imports of \`target.html\` and print the result to build.html.
+
+  The command
+    vulcanize -p "path/to/target/" /target.html
+  will inline the HTML Imports of \`target.html\`, treat \`path/to/target/\` as the webroot of target.html, and make all urls absolute to the provided webroot.
+
+  The command
+    vulcanize --exclude "path/to/target/subpath/" --exclude "path/to/target/subpath2/" target.html
+  will inline the HTML Imports of \`target.html\` that are not in the directory \`path/to/target/subpath\` nor \`path/to/target/subpath2\`.
+
+  If the \`--strip-excludes\` flag is used, the HTML Import \`<link>\` tags that point to resources in \`path/totarget/subpath\` and \`path/to/target/subpath2/\` will also be removed.
+
+  The command
+    vulcanize --inline-scripts target.html
+  will inline scripts in \`target.html\` as well as HTML Imports. Exclude flags will apply to both Imports and Scripts.
+  `,
+  raw: true}
+]
+
+    const options = commandLineArgs(optionDefinitions);
```

Comment:
The formatter is getting tripped up on something in this document and causing problems. I've had to revert a ton of egregiously horrible whitespace changes. I'll try and investigate what the cause is.